### ISSUE TITLE
language menu viewhelper won't find flag icons

### ISSUE THREAD

**Issue by featdd** [FluidTYPO3/vhs#854]
After I upgraded to 7.1 I recognized that the flag icons have moved and my language menu is rendered empty.

new path: typo3/sysext/core/Resources/Public/Icons/flags/
default: typo3/sysext/t3skin/images/flags/

Simply changing the path would break backwards compatibility.
Would catching the case make sense or is there a method to get the actual path?

(For workaround you can override the path with the flagPath argument)

**Comment by NamelessCoder** [comment]
Hi @featdd - afaik there is no core API to get this particular one, so I suggest the following (in pseudo-code because I'm a little busy - sorry about that) for the language menu ViewHelper:

```
protected function getCurrentCoreVersion() : float // returns for example 7.2 or 6.2
protected function getLanguageIconPath() : string // returns the right path based on what is returned from getCurrentCoreVersion().
```

Then simply call these from the render() method to get the path if it is not provided. The reason for these added methods is that this is exactly the methods I would need to refactor it into when adding unit tests and you'd be helping me by creating them that way - and if you want to help even more, write unit tests for it as well (but that's not a requirement) ;)

**Comment by featdd** [comment]
Will make a pull request in the next few days

**Comment by xf-** [comment]
Hi,
like i wrote in PR.
7.4 fails to render the image. The flagSrc is found.

**Comment by featdd** [comment]
fixed in #857
